docs/user-guide/creating-application/fluxcd.md

```diff
@@ -0,0 +1,118 @@
+
+{% hint style="info" %}
+### Additional Resources
+ [What is FluxCD? A Quick Guide to GitOps with FluxCD](https://devtron.ai/blog/what-is-fluxcd/)
+
+ [Choosing the Right GitOps Tool: ArgoCD vs.FluxCD](https://devtron.ai/blog/gitops-tool-selection-argo-cd-or-flux-cd/)
+
+{% endhint %}
+
+## Key features
+
+* Works with Devtron CI pipelines and other Devtron integrations.
+
+* Supports Helm deployment method only.
+
+## Installation
+
+{% hint style="warning" %}
+### Who Can Perform This Action?
+The user must have permissions to:
+  * Edit the ConfigMaps of 'default-cluster'
+  * Restart the pods
+{% endhint %}
+
+To enable deployments through GitOps via FluxCD or if you want to migrate your existing FluxCD application to Devtron, you need to enable two specific feature flags for the `default_cluster` in **Devtron** and then install the FluxCD controller on those clusters in which you want to deploy the FluxCD applications. To do so, follow the steps below:
+
+### Step 1: Enable Feature Flags
+
+1. Navigate to Devtron **Resource Browser**.
+
+      ![Figure 1: Navigating to Resource Browser](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/creating-application/fluxcd/fluxcd-resource-browser.jpg)
+
+2. Select the `default_cluster` to enable the feature flags.
+     
+      ![Figure 2: Selecting 'default_cluster'](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/creating-application/fluxcd/fluxcd-select-cluster.jpg)
+
+3. Go to Config & Storage → ConfigMap, and select `dashboard-cm` ConfigMap
+
+      ![Figure 3: Selecting 'dashboard-cm'](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/creating-application/fluxcd/fluxcd-select-dashboard-cm.jpg)
+
+4. Edit the `dashboard-cm` ConfigMap by clicking **Edit live manifest**.
+
+      * To enable deployments via FluxCD, check if the below entries are present in the ConfigMap (create one if it doesn't exist) and select **Apply changes**.<br>
+
+           ```yaml
+           FEATURE_FLUX_DEPLOYMENTS_ENABLE: " true"
+           FEATURE_LINK_EXTERNAL_FLUX_ENABLE: "true"
+           ```
+
+      ![Figure 4: Editing Live Manifest](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/creating-application/fluxcd/fluxcd-edit-live-manifest.jpg)
+
+      ![Figure 5: Adding Feature Flags](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/creating-application/fluxcd/fluxcd-add-flags.jpg)
+
+5. Restart the deployment: 
+
+      * For OSS Users:
+          1. Navigate to Devtron Resource Browser.
+
+          2. Select the cluster for which you have enabled the feature flags.
+
+          3. Click the **Terminal** tab.
+          
+          4. Restart the deployment using the following command:  
+
+               ```yaml
+               kubectl rollout restart deployment dashboard -n devtroncd 
```

Comment:
Will deleting the pod won't work? Navigating to cluster terminal to restart a rollout is not a good / user friendly flow